setup.cfg

```diff
@@ -40,17 +40,11 @@ install_requires =
 	waveorder==1.0.0rc0
 	pycromanager==0.19.2
 	click>=8.0.1
-	pyyaml>=5.4.1
-	tqdm>=4.61.1
-	opencv-python>=4.5.3.56 # candidate for deletion 1.0
 	natsort>=7.1.1
 	colorspacious>=1.1.2
 	pyqtgraph>=0.12.3
-	superqt>=0.2.4
 	napari-ome-zarr>=0.3.2 # drag and drop convenience
-	qtpy # napari best practice (PyQt5 and PySide2 wrapper)
 	napari[all] # uses pip-installed PyQt5 by default
-	imageio!=2.11.0,!=2.22.1 # matching napari PR #5168, candidate for removal after the upstream issue is fixed
```

Comment:
Can we delete it entirely? Or just remove the specific versions?

Reply:
See: https://github.com/mehta-lab/recOrder/pull/283#issuecomment-1319358403